CMakeCommon.cmake

```diff
@@ -370,6 +403,13 @@ function (ReadConfigJson)
         message (FATAL_ERROR "'languages' in config.json must be an array: ${error}")
     endif ()
 
+    # macOS codesigning
+    string (JSON codesignIdentityString ERROR_VARIABLE error GET "${json}" codesignIdentity)
```

Comment:
Don't deviate from the pattern without a good reason. Extend the extracted values in the `ReadConfigJson` command.